packages/core/admin/server/src/services/transfer/utils.ts

```diff
@@ -20,10 +13,17 @@ const hasValidTokenSalt = (): boolean => {
 /**
  * Checks whether data transfer features are enabled
  */
-const isDataTransferEnabled = (): boolean => {
+const isRemoteTransferEnabled = (): boolean => {
   const { utils } = getService('transfer');
 
-  return !utils.isDisabledFromEnv() && utils.hasValidTokenSalt();
+  // TODO v6: Remove this warning
+  if (env.bool('STRAPI_DISABLE_REMOTE_DATA_TRANSFER') !== undefined) {
+    strapi.log.warn(
+      'STRAPI_DISABLE_REMOTE_DATA_TRANSFER is no longer supported. Instead, set transfer.remote.enabled to false in your server configuration'
```

Comment:
We need to standardise these deprecation messages. They're inconsistent, for example in other places we've done `"[@strapi/strapi]: STRAPI_ENFORCE_SOURCEMAPS is now deprecated. You can enable sourcemaps by passing '--sourcemaps' to the build command."`

Thoughts?

Reply:
Sure, I can update them to match that message style, although in this case they're not just deprecated they're actually removed.

Reply:
Yeah I just meant the prefix of the package calling the error mainly... Do you think we need to document at some point the process of deprecation? 🤔

Reply:
I will add a contributor doc next week after some internal discussions on what we want to do, and then update this and/or the existing deprecation warnings at that time